analytics/src/main/java/com/rakuten/tech/mobile/analytics/Tracker.kt

```diff
@@ -0,0 +1,12 @@
+package com.rakuten.tech.mobile.analytics
+
+/** Event tracking dummy class. Please notice the proguard class path. */
+class Event(name: String, parameters: Map<String, Any>) {
+    fun track() {}
+}
+
+class RatTracker {
+    companion object {
+        fun event(name: String, parameters: Map<String, Any>) = Event(name, parameters)
```

Comment:
When I replaced the mock one with real analytics sdk, it's working 🎉  so I don't think we need to spend further effort here.

Reply:
Hmm, strange I got this error when I tried:
```
java.lang.NoSuchFieldError: No field Companion of type Lcom/rakuten/tech/mobile/analytics/RatTracker$Companion
```

Which makes sense because the kotlin code will be compiled to call `RatTracker.Companion.event` which doesn't exist in the real analytics package.

Did you run `./gradlew clean` before building? Or maybe I'm doing something different 🤔 

Seems it should also work if you change it to an `object` class instead:

```
object RatTracker {
  @JvmStatic
  fun event()
}
```

Reply:
updated